stable/nginx-ingress/values.yaml

```diff
@@ -5,7 +5,7 @@ controller:
   name: controller
   image:
     repository: quay.io/kubernetes-ingress-controller/nginx-ingress-controller
-    tag: "0.15.0"
+    tag: "0.17.1"
```

Comment:
please update the README and Chart.yaml as well (both chart version and the appVersion)

Reply:
also the CLA needs signing - see comments from the robot.